src/main/scala/org/apache/spark/sql/execution/benchmark/TPCDSDatagen.scala

```diff
@@ -714,5 +714,6 @@ object TPCDSDatagen {
       datagenArgs.filterOutNullPartitionValues,
       datagenArgs.tableFilter,
       datagenArgs.numPartitions.toInt)
+    spark.stop()
```

Comment:
Hi, @maropu . Can we add `spark.stop()` here since your repository is a best example for Spark apps? :)

Reply:
Yea, thanks for you comment, Dongjoon! Sure, I'll update it.